Mathlib/Geometry/Euclidean/Incenter.lean

```diff
@@ -244,6 +244,20 @@ terms of `ExcenterExists`. -/
 lemma excenterExists_singleton [Nat.AtLeastTwo n] (i : Fin (n + 1)) : s.ExcenterExists {i} :=
   (s.sum_excenterWeightsUnnorm_singleton_pos i).ne'
 
+/-- The barycentric coordinates of the incenter are less than `2⁻¹` (thus, it lies closer on an
```

Comment:
It's initially confusing that a statement about the incenter is spelled using the language of the excenter. I understand that good reasons why this is so but I wonder what we could do to help. One option would be to add:
```lean
abbrev incenterWeights : Fin (n + 1) → ℝ := s.excenterWeights ∅
```
just below `excenterWeights` above and then to restate and rename this lemma to use `incenterWeights`. I note that this would be consistent with the fact that we already have [Affine.Simplex.incenter](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Geometry/Euclidean/Incenter.html#Affine.Simplex.incenter).

Another option (which seems less appealing to me) would be to leave the code alone but call this out really explicitly in the doc string.